### PR TITLE
gh-95273: Condense sqlite3 executescript example

### LINKED ISSUE #95273: Improve the sqlite3 reference
Improve the sqlite3 reference, using [Diataxis as a guideline](https://diataxis.fr/reference/). A lot of work has already been done here, with great help from Alex Waygood and CAM Gerlach. PRs so far[^1] have focused on...:

- [Be accurate](https://diataxis.fr/reference/#be-accurate)
- [Do nothing but describe](https://diataxis.fr/reference/#do-nothing-but-describe); making the wording more to the point

<details>
<summary>Previous work over the last few years</summary>

Both of these have been improved greatly by the following issues and PRs, amongst others:
- gh-94630
- gh-95235
- gh-95235
- gh-94940
- gh-94321
- gh-94628
- gh-94017
- gh-93890
- gh-93925
- gh-87260
- gh-89018
- gh-92780
- gh-90923
- gh-30660
- gh-29326
- gh-29200
- gh-29219
- gh-28238
- gh-26220
- gh-25003
- gh-23159
- gh-23965
- consistency: gh-31772
- consistency: gh-24746

</details>

There are remaining issues, though:

- [x] Be accurate:
    - #95379
    - #95523
    - #95530
- [x] [Be consistent](https://diataxis.fr/reference/#be-consistent): Most of the reference docs for sqlite3 functions and methods use the imperative mood. Some of the methods use other wordings. Suggesting to normalise, and use the imperative mood across the reference section.
  Improved with:
    - #95274 
    - #95525
    - Remove duplicate links
- [x] [Respect the structure of the machinery](https://diataxis.fr/reference/#respect-the-structure-of-the-machinery): Arrange the sections of reference to follow the architecture of the module. This is pretty good already, but I believe we may benefit from some adjustments, in particular:
    - [x] Consistently group attributes and methods
    - #95626
    - #95430
    - Align wording in function and method docstrings with documentation
- [x] [Do nothing but describe](https://diataxis.fr/reference/#do-nothing-but-describe): Avoid digressions and discussions, or even better, extract them to separate sections. Remove and or simplify examples.
    - #95351
    - #95383
    - gh-95840

[^1]: There have been too many PRs to list; some of them were done before Diataxis was even discussed.

### PR DESCRIPTION
**Pull request by erlend-aasland**
* Issue: gh-95273

### CODE REVIEW

**Comment by erlend-aasland** [comment]
Also, the current example is faulty bco. lack of transaction handling: there is no commit; the connection is closed with a pending transaction.

File: Doc/library/sqlite3.rst [hunk 1]
```diff
@@ -1007,9 +1007,15 @@ Cursor Objects
 
       *sql_script* must be a :class:`string <str>`.
 
-      Example:
+      Example::
 
-      .. literalinclude:: ../includes/sqlite3/executescript.py
+         cur.executescript("""
```
Comment (erlend-aasland) [inline Doc/library/sqlite3.rst:1013]: In contrast with the tutorial, examples in the reference can assume familiarity. I believe it is ok to drop the import/connect setup, but I can be convinced to include it.

Comment (CAM-Gerlach) [reply Doc/library/sqlite3.rst:1013]: I'm generally sympathetic to the argument of keeping them minimal and assuming a basic level of familiarity. On the other hand, Reference docs should also prioritize clarity above all, and at least to me it isn't great form to leave what `cur` is completely implicit—at least a comment to the affect of `# cur is a sqlite3.Cursor object` would help alleviate that.

**Review by CAM-Gerlach — COMMENTED** [review]
LGTM, aside from one comment responding to yours

File: Doc/library/sqlite3.rst (hunk 1 above)
Comment (erlend-aasland) [reply Doc/library/sqlite3.rst:1013]: Yes, I was worried about this. Let's introduce a comment.

Comment (erlend-aasland) [reply Doc/library/sqlite3.rst:1013]:
```suggestion
         # cur is an sqlite3.Cursor object
         cur.executescript("""
```

**Comment by erlend-aasland** [comment]
Thanks for the review, CAM!

**Pull request merged**